src/service/xhr-impl.js

```diff
@@ -47,6 +47,13 @@ const allowedMethods_ = ['GET', 'POST'];
 /** @private @const {!Array<function(*):boolean>} */
 const allowedJsonBodyTypes_ = [isArray, isObject];
 
+/** @private @enum {number} Allowed fetch responses. */
+const allowedFetchTypes_ = {
+  document: 1,
```

Comment:
Makes sense.  But in this case, it's really just being used as a Set -- we just want to quickly look up whether a requested response type is allowed -- see `#fetchPolyfill`.  (I was told not to use Set itself b/c it wasn't well supported.)  So the keys are defined by the fetch API, not by us.